### ISSUE TITLE
Whitespace in arguments not passing to WP-CLI properly

### ISSUE THREAD

**Issue by aaemnnosttv** [10up/wp-local-docker-v2#6]
Example using a fresh install of WP

```
$ 10updocker wp post list              
+----+--------------+-------------+---------------------+-------------+
| ID | post_title   | post_name   | post_date           | post_status |
+----+--------------+-------------+---------------------+-------------+
| 1  | Hello world! | hello-world | 2018-06-05 21:24:58 | publish     |
+----+--------------+-------------+---------------------+-------------+
```

Using the DB search command

```
$ 10updocker wp db search                
usage: wp db search <search> [<tables>...] [--network] [--all-tables-with-prefix] [--all-tables] [--before_context=<num>] [--after_context=<num>] [--regex] [--regex-flags=<regex-flags>] [--regex-delimiter=<regex-delimiter>] [--table_column_once] [--one_line] [--matches_only] [--stats] [--table_column_color=<color_code>] [--id_color=<color_code>] [--match_color=<color_code>]

$ 10updocker wp db search 'hello world'
Error: Couldn't find any tables matching: world
```

WP-CLI is getting `world` as the second argument to the command, when it should be part of the first.

The command works as expected when adding extra quotes.

```
$ 10updocker wp db search '"hello world"'
wp_options:option_value
3:Hello World Project
wp_options:option_value
138:Hello World Project
wp_posts:post_title
1:Hello world!
```

**Comment by eugene-manuilov** [comment]
@aaemnnosttv, created a PR to fix this issue. See here: #60

**Comment by jeffpaul** [comment]
Closed by #60
